src/pages/events/2020/03/sugcon.md

```diff
@@ -0,0 +1,22 @@
+---
+templateKey: 'event-page'
+eventId: 
+title: SUGCON - 26 & 27 March 2020
+sup: The Sitecore User Group Conference (SUGCON) brings the Sitecore user community together and provides the perfect platform to gain knowledge and get inspired by the potential within Sitecore Experience Cloud, Experience Commerce, Content Hub and more. After many very successful events, we are expecting over 500 attendees to attend the event..

```

Comment:
potential of

Reply:
Not sure what you mean - copied text from Sugcon site?

Reply:
Hah, it's a grammar nit-pick anyway